### PR TITLE
fix(virtual-background-guide): updated virtual background guide to support v2 changes

### PR DESCRIPTION
**Pull request by ravindra-dyte**
## Description

Updated virtual background guide to support v2 changes.

### CODE REVIEW

File: docs/guides/capabilities/video/add-virtual-background.mdx [hunk 1]
```diff
@@ -40,13 +40,37 @@ import DyteVideoBackgroundTransformer from '@dytesdk/video-background-transforme
 
 If you installed the package using a script tag, the `DyteVideoBackgroundTransformer` will be automatically available for use once the installation process is complete.
 
-To begin using the Dyte Background Transformer, you need to initialize it by calling the `init` method. Follow the steps below to initialize the transformer:
+2.x versions of DyteVideoBackgroundTransformer use their own rendering mechanism thus require you to disable the default per frame canvas rendering.
 
 ```js
-const videoBackgroundTransformer = await DyteVideoBackgroundTransformer.init();
+await meeting.self.setVideoMiddlewareGlobalConfig({ disablePerFrameCanvasRendering: true });
```
Comment (palashgo) [inline docs/guides/capabilities/video/add-virtual-background.mdx:46]: We don't need this now do we

Comment (ravindra-dyte) [reply docs/guides/capabilities/video/add-virtual-background.mdx:46]: 2.0.1 & 2.0.2 still require this line to be added by the devs. 2.0.3 onwards, we are internally doing this.

I have kept it here for reference. Also calling this method multiple times is safe and doesn't cause any side effects.

**Review by ishita1805 — APPROVED** [review]
lgtm

File: docs/guides/capabilities/video/add-virtual-background.mdx (hunk 1 above)
Comment (ishita1805) [reply docs/guides/capabilities/video/add-virtual-background.mdx:46]: maybe add a note saying not required post 2.0.3?

Comment (ravindra-dyte) [reply docs/guides/capabilities/video/add-virtual-background.mdx:46]: It is added exclusively to highlight it as a breaking change for any custom middlewares that our clients might have.

If any middleware is broken, migration could be done by referring to company branding sample.

Default middleware design: https://github.com/dyte-in/streams-middleware-samples/blob/main/video/company-branding.js
Design if per frame rendering is disabled: https://github.com/dyte-in/streams-middleware-samples/blob/main/video/middleware-with-per-frame-rendering-disabled/company-branding.js

**Pull request merged**